src/pages/SettingsPage/PreferencesPage.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import {View} from 'react-native';
+import RNPickerSelect from 'react-native-picker-select';
+import {withOnyx} from 'react-native-onyx';
+import PropTypes from 'prop-types';
+
+import HeaderWithCloseButton from '../../components/HeaderWithCloseButton';
+import {redirect} from '../../libs/actions/App';
+import HeaderGap from '../../components/HeaderGap';
+import ROUTES from '../../ROUTES';
+import ONYXKEYS from '../../ONYXKEYS';
+import styles from '../../styles/styles';
+import Text from '../../components/Text';
+import Icon from '../../components/Icon';
+import NameValuePair from '../../libs/actions/NameValuePair';
+import CONST from '../../CONST';
+import {DownArrow} from '../../components/Icon/Expensicons';
+
+
+const propTypes = {
+    // The chat priority mode
```

Comment:
existing, not exactly sure what it's meant for so not sure what to update it to

Reply:
I suggest removing it, or updating the comment to be `// see priorityModes below`

Reply:
Just to chime in here... `propTypes` are _usually_ just a simple description of the prop so I think what we have here is OK (of course it's always nice to be more descriptive). However, having a `propType` comment ask you to "look below" does not seem correct since we may use them in the future e.g. for generating docs or alongside something like storybook where they would be isolated from the rest of the code.